test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -999,10 +999,19 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 	})
 })
 
-func installNvidiaDriversDaemonSet(namespace string) {
+func installNVIDIADriversDaemonSet(f *framework.Framework) {
 	ginkgo.By("Add daemonset which installs nvidia drivers")
-	// the link differs from one in GKE documentation; discussed with @mindprince this one should be used
-	framework.RunKubectlOrDie(namespace, "apply", "-f", "https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/daemonset.yaml")
```

Comment:
are we removing usage of external calls to `kubectl`?

Reply:
currently , we use kubectl in many places , it is difficult to remove kubectl , i  recommend f.ClientSet just here.

Reply:
I don't know if this is correct.